src/libsystemd-network/dhcp-identifier.h

```diff
@@ -17,6 +17,7 @@ typedef enum DUIDType {
         DUID_TYPE_EN        = 2,
         DUID_TYPE_LL        = 3,
         DUID_TYPE_UUID      = 4,
+        DUID_TYPE_CUSTOM    = 5,
```

Comment:
Hmm??? I cannot find such DUID type. Which RFC did you see?

Reply:
Yeah, I understand the motivation, but the enum is for the first 2bytes of DUID. We should not mix them.
-> #28918.

Reply:
That is precisely we want to avoid according to the  https://github.com/systemd/systemd/issues/26745. Because of the type the DHCP client is not to work as is supposed to

Reply:
Sorry, I cannot follow.

I mean the enum should not be used for condition that how we handle `DUIDRawData=`. We should always assign `DUIDType=` to the first two bytes of DUID.

Note, with #28918, the requested DUID can be set by
```
DUIDType=0
DUIDRawData=ab:11:f9:2a:c2:77:29:f9:5c:00
```

Reply:
The RFE is not to set `DUIDType itself` .  Here its  spitted to fit like that. Now user had to split every time and fill the type and the rawdata. They cannot simply use the rawdata